docs/dogfood/index.md

```diff
@@ -0,0 +1,143 @@
+# Dogfooding
+
+This guide explains how to [dogfood](https://www.techopedia.com/definition/30784/dogfooding) coder for employees at Coder.
+
+## How to 
+
+The following explains how to do certain things related to dogfooding.
+
+### Dogfood using Coder's Deployment
+
+1. Go to [https://dev.coder.com](https://dev.coder.com/) 
+    - If you see a dialog/pop-up, hit "Cancel"
+2. Sign in with GitHub
+3. Clone the repo: `git clone git@github.com:coder/coder.git` 
```

Comment:
Is this the suggestion to dogfood this locally?

Reply:
I believe that's what it was originally. I have no opinions here so happy to go whichever direction is recommended

Reply:
I think we should promote the usage of dev.coder.com primarily. There are few reasons (in my experience) that I've had to swap to local, and it's generally been a pain.